### PR TITLE
Fix Pagination initial state and add specs

### PR DESCRIPTION
**Pull request by alextrastero**

### CODE REVIEW

File: src/Pagination.js [hunk 1]
```diff
@@ -6,23 +6,30 @@ import PaginationButton from './PaginationButton';
...
+      activePage: activePage > 0 && activePage <= items
+        ? activePage : 1
     };
 
     this.renderButtons = this.renderButtons.bind(this);
     this._onClick = this._onClick.bind(this);
   }
 
+  componentWillReceiveProps ({ activePage }) {
+    const { items } = this.props;
```
Comment (isaiah) [inline src/Pagination.js]: I think the `items` props might be updated as well.

Comment (alextrastero) [inline src/Pagination.js]: yeah, good point

Comment (alextrastero) [inline src/Pagination.js]: For now, I think we can merge this as is. I can't think of a scenario where the items would update externally.

Comment (alextrastero) [inline src/Pagination.js]: cc @isaiah

**Comment by isaiah** [comment]
rebase please.

**Comment by isaiah** [comment]
@alextrastero ping?

**Comment by alextrastero** [comment]
Yeah, need to revisit this

**Pull request closed without merging**